src/common/navigation.yaml

```diff
@@ -54,6 +54,7 @@ sections:
           - path: /network
           - path: /stacking
           - path: /local-development
+          - path: /configurations
```

Comment:
could we rename the file so it is more descriptive in the navigation? I'd propose something like `node-configuration`

Reply:
update: with this looking more like a reference, maybe it is better to name it `stacks-node` or similar (so it would be accessible as: "references > stacks node"). Would suggest a slightly broader scope, but might be good to have a space for node references in general

Reply:
[What do you think of this?](https://3000-a973afd8-e39b-4e4c-9108-9aeb922360b9.ws-us02.gitpod.io/) (Using my fancy gitpod)

Reply:
👍